consensus/quorum/quorum.go

```diff
@@ -273,7 +273,7 @@ func (s *cIdentities) SignersCount(p Phase) int64 {
 	}
 }
 
-func (s *cIdentities) SubmitVote(
+func (s *cIdentities) submitVote(
```

Comment:
need a better description on how to use this function, can you add some comments.

Reply:
Done.